charts/nexus-iq/tests/ingress_test.yaml

```diff
@@ -0,0 +1,85 @@
+suite: ingress
+templates:
+  - ingress.yaml
+tests:
+  - it: renders with defaults
+    set:
+      ingress:
+        enabled: true
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Ingress
+      - equal:
+          path: apiVersion
+          value: networking.k8s.io/v1
```

Comment:
nice to see some sort of a test! Wild.

Reply:
This will help us a lot! Thanks for putting all of this together.